crates/mitex-web-wasm/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "mitex-wasm-web"
```

Comment:
Can we merge `mitex-web-wasm` and `mitex-typst` into `mitex-wasm`?
I feel okay on build typst plugin with 
```bash
cargo build --bin mitex-wasm --feature typst-plugin
```
And web wasm with

```bash
wasm build --feature web
```